tensorflow/python/keras/utils/generic_utils_test.py

```diff
@@ -310,6 +310,21 @@ def from_config(cls, config):
     self.assertEqual(new_layer.units, 3)
     self.assertIs(new_layer.units.fn, serializable_fn)
 
+  def test_serialize_type_object_initializer(self):
+    layer = keras.layers.Dense(
+        1,
+        kernel_initializer=keras.initializers.ones,
+        bias_initializer=keras.initializers.zeros)
+    config = keras.layers.serialize(layer)
+    self.assertEqual(
+        config['config']['bias_initializer'],
```

Comment:
Actually, in certain conditions we will stick a dtype into the config, causing this to fail. Could you update this to 

config['config']['bias_initializer']['class_name'], 'Zeros'

That should be more robust and allow us to merge this.